packages/datadog-plugin-google-cloud-pubsub/test/index.spec.js

```diff
@@ -21,9 +20,9 @@ describe('Plugin', () => {
       delete process.env.PUBSUB_EMULATOR_HOST
     })
     afterEach(() => {
-      return agent.close()
+      return agent.close({ ritmReset: false })
     })
-    withVersions(plugin, '@google-cloud/pubsub', version => {
+    withVersions('google-cloud-pubsub', '@google-cloud/pubsub', version => {
```

Comment:
We should test that all spans are finished when the lease is cleared since this is not currently the case and no test are failing.